try{
    //Do something
}catch{
    console.log("Err");
}

try{
    //Do something
}catch(err){
    console.log("Error "+err);
}